src/test/java/api/support/builders/HoldingBuilder.java

```diff
@@ -85,7 +93,18 @@ public HoldingBuilder withCallNumber(String callNumber) {
       this.instanceId,
       this.permanentLocationId,
       this.temporaryLocationId,
-      callNumber
+      callNumber,
+      this.holdingsItems
+    );
+  }
+
+  public HoldingBuilder withHoldingsItems(JsonArray holdingsItems) {
```

Comment:
This does not appear to be used. Is this a hangover from the confusion caused by the GraphQL metadata?